src/matchers/mock/toBeRequested.ts

```diff
@@ -0,0 +1,28 @@
+import { waitUntil, enhanceError } from '../../utils'
+import { runExpect } from '../../util/expectAdapter'
+
+function toBeRequestedFn(received: WebdriverIO.Mock, options: ExpectWebdriverIO.CommandOptions = {}): any {
```

Comment:
Do you think it worst calling `toBeRequestedTimesFn` assuming that it accepts `{ gte: 1 }` ?

Reply:
So instead of:

```js
expect(mock).toBeRequestedTimes(1)
```

to write

```
expect(mock).toBeRequestedTimes({ eq: 1 })
// or 
expect(mock).toBeRequestedTimes({ gte: 2 })
```

Sounds good to me, just want to make sure I understood correctly.

Reply:
is it possible to have both `expect(mock).toBeRequestedTimes({ eq: 1 })` and `expect(mock).toBeRequestedTimes(1)`, and also add `expect(mock).toBeRequestedTimes({ gte: 2 })`, `expect(mock).toBeRequestedTimes({ lt: 2 })`, etc?

Reply:
Yes, let me add it. Thanks for the feedback!